### PR TITLE
[bitnami/postgresql-ha] Add labels to volume claim template

### PR DESCRIPTION
**Pull request by sl4dy**
Allow addition of labels to volume claim template in the StatefulSet.

Signed-off-by: Milan Sladky <milan.sladky@sentinelone.com>

### Description of the change

Allow addition of labels to volume claim template in the StatefulSet.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for your PR, in order to make the GH actions work and to be able to verify the changes, could you rebase from `master`? Thank you

**Comment by sl4dy** [comment]
@carrodher @fmulero rebased

File: bitnami/postgresql-ha/Chart.yaml [hunk 1]
```diff
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 9.2.8
+version: 9.2.9
```
Comment (fmulero) [inline bitnami/postgresql-ha/Chart.yaml:30]: We are adding a new feature, so according to [semver](https://semver.org/), we should release the version `9.3.0`

```suggestion
version: 9.3.0
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Thanks a lot @sl4dy !!!

Sorry to be a little bit picky, but could you take a glance to my comment about the version? All the rest look perfect

**Comment by sl4dy** [comment]
@fmulero done

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**